lib/PHPCfg/Op/Expr/Assertion.php

```diff
@@ -17,6 +17,8 @@
 
 class Assertion extends Expr
 {
+    public Operand $expr;
+
     public Operand $read;
```

Comment:
Drop this one? Looks like only expr is actually in use.

Reply:
done